docker-compose.yml.example

```diff
@@ -0,0 +1,18 @@
+version: '3.1'
```

Comment:
Missing a database?

Reply:
I thought the same thing, but I haven't had chance to get to the installation page. Would MariaDB work here?

Reply:
MariaDB can be added yeah